### DISCUSSION TITLE
Configure Betaflight through ELRS Wifi

### DISCUSSION THREAD

**Discussion by mooiweertje** [ExpressLRS/ExpressLRS#1312] [Ideas]
Wouldn't it be great to be able to configure  Betafliflight/iNav via WiFi of the ESP32?

**Comment by AlessandroAU** [comment]
Perhaps you haven't seen this :) https://github.com/ExpressLRS/ExpressLRS/pull/1274

**Reply by mooiweertje** [reply]
I missed that one. Thanks!
